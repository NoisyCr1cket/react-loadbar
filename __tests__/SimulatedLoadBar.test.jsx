import React from 'react'
import { SimulatedLoadBar } from '../dist/index'
import renderer from 'react-test-renderer'

test('SimulatedLoadBar renders properly', () => {
    const cmpt = renderer.create(
        <SimulatedLoadBar />
    )

    const tree = cmpt.toJSON()
    expect(tree).toMatchSnapshot()
})