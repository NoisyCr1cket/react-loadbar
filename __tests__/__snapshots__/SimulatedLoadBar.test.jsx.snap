// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimulatedLoadBar renders properly 1`] = `
<div
  className="loadbar-wrap"
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <div
    className="loadbar-spinner"
    style={Object {}}
  />
  <div
    className="loadbar-progress"
    style={
      Object {
        "width": "1%",
      }
    }
  />
</div>
`;
